.gitignore

```diff
@@ -28,5 +28,7 @@ override.tf.json
 # Include tfplan files to ignore the plan output of command: terraform plan -out=tfplan
 # example: *tfplan*
 
+.terraform.lock.hcl
```

Comment:
Specifiek moeten deze lock files in de modules/ directory niet meegecommit worden, niet per se in het hele project, toch? Stel dat we voorbeelden gaan toevoegen.

Reply:
Bekijk ik wel wanneer ik met de voorbeelden bezig ga.